### PR TITLE
Added a Security-chaos-Engineering in Hindi.

### PR DESCRIPTION
**Pull request by abhay-raj19**
Signed-off-by : abhay-raj19 <rajpootabhay423@gmail.com>
### Describe your changes
Localised a security chaos engineering document in hindi.

### Related issue number or link (ex: `resolves #issue-number`)
#1986 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/security-chaos-engineering.md
Comment (jayesh-srivastava) [inline content/hi/security-chaos-engineering.md:12]: Instead of 'जिससे प्रणाली की क्षमता के विषय में विश्वास बढ़ता है' we can write 'जिससे प्रणाली की क्षमता में विश्वास बढ़ता है'

File: content/hi/security-chaos-engineering.md
Comment (jayesh-srivastava) [inline content/hi/security-chaos-engineering.md:14]: Extra lines between करते and हुए . Please remove

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/security-chaos-engineering.md
Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md]: Changes made as per the suggestions. 
Thanks @jayesh-srivastava for reviewing up 💙.

File: content/hi/security-chaos-engineering.md
Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md]: Done.

File: content/hi/security-chaos-engineering.md
Comment (jayesh-srivastava) [inline content/hi/security-chaos-engineering.md:30]: If we are pinning down on अव्यवस्था as the translation for chaos, we should be using it everywhere.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/security-chaos-engineering.md
Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md]: Done the changes as mentioned above.

Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md]: @jayesh-srivastava now pls see to it.

Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md]: @jayesh-srivastava please re-review it again now I have corrected all those changes suggested.

**Comment by seokho-son** [comment]
Hi @jayesh-srivastava 
Could you check the latest modification resolves your concern ?

**Comment by abhay-raj19** [comment]
Hi @anubha-v-ardhan can you check and review up my changes as suggested .

**Comment by jihoon-seo** [comment]
Friendly pinging Hindi localization approvers @Garima-Negi @sayantani11 @anubha-v-ardhan @jayesh-srivastava

File: content/hi/security-chaos-engineering.md
Comment (jayesh-srivastava) [inline content/hi/security-chaos-engineering.md:38]: You can use hindi to mention SRE, devops and devsecops with their corresponding English words in parenthesis and both can point to the url.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/security-chaos-engineering.md
Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md]: @jayesh-srivastava now pls check the content i have updated as told by you

File: content/hi/security-chaos-engineering.md
Comment (jayesh-srivastava) [inline content/hi/security-chaos-engineering.md:38]: In parenthesis you can also add the english word.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/security-chaos-engineering.md
Comment (abhay-raj19) [reply content/hi/security-chaos-engineering.md:38]: यह दृष्टिकोण [एसआरईएस(SREs)](/site-reliability-engineering/), [डेवऑप्स(DevOps)](/devops/) और [डेवसेकओप्स(DevSecOps)](/devsecops/) इंजीनियरों को सिस्टम में आत्मविश्वास बनाने,

@jayesh-srivastava like this or some thing diffrent .

**Review by jayesh-srivastava — APPROVED** [review]
Lgtm. Thanks

**Comment by seokho-son** [comment]
Hi @jayesh-srivastava
Thank you for reviewing this PR.
Since this PR requires at least two approvals from hi approvers, I hope to check which approver is going to finalize this PR.
(considering the pending status of this PR)

Could you help assigning an appropriate approver for this PR? Or does Hindi team needs any help from maintainers?

**Comment by abhay-raj19** [comment]
@Garima-Negi pls do riview it ASAP.

**Pull request closed without merging**
